phpBB/language/en/help/faq.php

```diff
@@ -183,4 +184,17 @@
 	'HELP_FAQ_USERSETTINGS_SERVERTIME_QUESTION'	=> 'I changed the timezone and the time is still wrong!',
 	'HELP_FAQ_USERSETTINGS_TIMEZONE_ANSWER'	=> 'It is possible the time displayed is from a timezone different from the one you are in. If this is the case, visit your User Control Panel and change your timezone to match your particular area, e.g. London, Paris, New York, Sydney, etc. Please note that changing the timezone, like most settings, can only be done by registered users. If you are not registered, this is a good time to do so.',
 	'HELP_FAQ_USERSETTINGS_TIMEZONE_QUESTION'	=> 'The times are not correct!',
+
+	'HELP_FAQ_WEBPUSH_GENERAL_ANSWER'	=> 'Make sure this forum is allowed to send notifications in your browser settings. Also, verify that your device’s system settings permit notifications from your web browser or app. Some browsers deliver notifications even when closed, whilst others only do so when the browser is open. <a href="https://caniuse.com/push-api" target="_blank">View this table for browser support information.</a>',
```

Comment:
We don't use `target=_blank`, or?

Reply:
Then how do we open external site in new window?

Reply:
Afaik we just don't. 😀 Users should do it with their browser. 

Or via JS?

Reply:
If you search for it you'll see it still used here and there.

Reply:
AFAIK `_target="blank"` shouldn't be an issue anymore as since ~2021 browsers implicitly set `rel=noopener` for any `target="_blank"` link in according to HTML specs change. Sources:
https://stackoverflow.com/a/4964223
https://mathiasbynens.github.io/rel-noopener/
https://github.com/whatwg/html/pull/4330